validators/terravaloper1f337thg2cxu6yny9hquec42p4s2a27awyu8cgf/profile.json

```diff
@@ -0,0 +1,13 @@
+{
+  "contact": {
+    "email": "contact@lunanauts.net",
+    "twitter": "@Lunanauts_Lunc"
+  },
+    "notifications": { // notification settings

```

Comment:
Hey @lunanauts!

Please remove the comments in your `profile.json` so that it can adhere to JSON formatting.

Thanks.

Reply:
hi changes have been made, thanks.